src/compute/protocol.rs

```diff
@@ -224,6 +250,11 @@ pub struct ServerCreate {
         skip_serializing_if = "Vec::is_empty"
     )]
     pub block_devices: Vec<BlockDevice>,
+    #[serde(
+        skip_serializing_if = "Option::is_none",
+        serialize_with = "config_drive_string_to_bool"
+    )]
+    pub config_drive: Option<bool>,
```

Comment:
I wonder if we could reuse `has_config_drive` for both serialization and deserialization. That would involve calling `unwrap` on an `Option` though..